mlflow/pyfunc/__init__.py

```diff
@@ -691,12 +727,14 @@ def _load_model_or_server(model_uri: str, env_manager: str):
 
     :param model_uri: The uri of the model.
     :param env_manager: The environment manager to load the model.
+    :param model_config: The model configuration to use by the model, only if the model
+                             accepts it.
     :return: A _ServedPyFuncModel for non-local ``env_manager``s or a PyFuncModel otherwise.
     """
     from mlflow.pyfunc.scoring_server.client import ScoringServerClient, StdinScoringServerClient
 
     if env_manager == _EnvManager.LOCAL:
-        return load_model(model_uri)
+        return load_model(model_uri, model_config=model_config)
```

Comment:
With above update, we can read load_config directly from the model's file, then no need to update this.

Reply:
It is required. Take a look at how `mlflow.evaluate()` works.